native/packages/react-native-bpk-component-navigation-bar/readme.md

```diff
@@ -79,13 +79,23 @@ import BpkNavigationBar from 'react-native-bpk-component-navigation-bar'
 
 #### Props
 
-| Property       | PropType               | Required | Default Value |
-| -------------- | ---------------------- | -------- | ------------- |
-| title          | oneOf(string, element) | true     | -             |
-| theme          | object                 | false    | null          |
-| leadingButton  | button element         | false    | null          |
-| trailingButton | button element         | false    | null          |
-| subtitleView   | element                | false    | null          |
+| Property       | PropType                             | Required | Default Value |
+| -------------- | ------------------------------------ | -------- | ------------- |
+| title          | oneOf(string, titleWithIcon element) | true     | -             |
```

Comment:
I think that instead of `titleWithIcon element`, we should say `titleWithIcon shape`. `element` is a bit ambiguous with the element type IMO.